indy_node/test/upgrade/test_broken_connection_control_tool.py

```diff
@@ -0,0 +1,69 @@
+import dateutil
+import dateutil.tz
+import pytest
+
+from datetime import datetime, timedelta
+from copy import deepcopy
+from indy_common.constants import START
+from indy_node.test.upgrade.conftest import patch_packet_mgr_output, EXT_PKT_NAME, EXT_PKT_VERSION
+
+from indy_node.server.upgrader import Upgrader
+from indy_node.test.upgrade.helper import sdk_ensure_upgrade_sent, bumpedVersion
+from plenum.test.helper import randomText
+
+from stp_core.common.log import getlogger
+
+delta = 2
+logger = getlogger()
+
+
+@pytest.fixture(scope="module")
+def tconf(tconf):
+    old_delta = tconf.MinSepBetweenNodeUpgrades
+    tconf.MinSepBetweenNodeUpgrades = delta
+    yield tconf
+    tconf.MinSepBetweenNodeUpgrades = old_delta
+
+
+@pytest.fixture(scope='function')
+def validUpgrade(nodeIds, tconf, monkeypatch):
```

Comment:
Why can't we use existing `validUpgrade` fixture and update its output if needed?

Reply:
existing validUpgrade parametrized with different kind of versions. But probably that doesn't matter. Will change it